providers/aws/resources/aws.lr

```diff
@@ -797,6 +798,20 @@ private aws.iam.user @defaults("arn name") {
   loginProfile() aws.iam.loginProfile
 }
 
+// AWS IAM instanceProfile
+private aws.iam.instanceProfile @defaults("arn instanceProfileId") {
+  // ARN of the IAM instanceProfile
+  arn string
+  // Time when instanceProfile was created
+  createDate time
+  // ID of the IAM instanceProfile
+  instanceProfileId string
+  // name of the IAM instanceProfile
+  instanceProfileName string
+  // Tags for the IAM user
```

Comment:
This isn't tags for the instance profile? Just checking. :)

Reply:
Good call, thanks @misterpantz !